integration/pmem/pmem_test.sh

```diff
@@ -40,7 +40,6 @@ fi
 
 init() {
 	${dir_path}/../../integration/kubernetes/init.sh
-	docker run -d -p 5000:5000 --restart=always --name registry registry:2
```

Comment:
@devimc no, I did intend to use `docker rm`, not `docker image rm`. It's just that from the line reference here, I take there is no need to run the registry any more, which confuses me because the container should have been removed.

Reply:
@devimc I don't feel this is resolved. Did you have issues when the line referenced here was present? Also, on the VFIO CI, the registry container wouldn't even have been launched because we don't run the k8s tests there.

Reply:
wait, are we running the pmem tests on any CI in the first place?

Reply:
@Jakob-Naucke yes we are running pmem tests on a CI and we test this PR locally and is fixing the pmem CI

Reply:
> Did you have issues when the line referenced here was present? 

yes, because that port was already in use

> Also, on the VFIO CI, the registry container wouldn't even have been launched because we don't run the k8s tests there.

does VFIO CI need a registry?  I don't think so

Reply:
I thought the VFIO CI ran the pmem tests, but it doesn't seem so. Let me TAL...

Reply:
Ah. It's because the pmem tests also run our k8s init. Let me resolve this conversation. Sorry for the noise! But I wanted to make sure I'd understand the change.